raco/backends/myria/myria.py

```diff
@@ -1663,6 +1706,7 @@ def fire(self, expr):
     rules.OneToOne(algebra.Difference, MyriaDifference),
     rules.OneToOne(algebra.OrderBy, MyriaInMemoryOrderBy),
     rules.OneToOne(algebra.Sink, MyriaSink),
+    DeBroadcastToSelect()
```

Comment:
Do we actually need this rule given the `MoveDeBroadcast` rule, which pushes `DeBroadcast` ops all the way down to scans, and then sets `overrideWorkers` on those scans to `[0]`? The `DeBroadcastToSelect` rule seems very hackish by comparison.